hooks/open-telemetry/pkg/otel.go

```diff
@@ -2,165 +2,53 @@ package otel
 
 import (
 	"context"
-	"encoding/json"
-	"errors"
-	"fmt"
-	"sync"
 
-	of "github.com/open-feature/go-sdk/pkg/openfeature"
+	"github.com/open-feature/go-sdk/pkg/openfeature"
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/trace"
 )
 
 const (
-	// AttributeFlagKey trace attribute name for the evaluated flag key
-	AttributeFlagKey = "feature_flag.flag_key"
-	// AttributeProviderName trace attribute name for the provider used for evaluation
-	AttributeProviderName = "feature_flag.provider_name"
-	// AttributeEvaluatedVariant trace attribute name for the variant returned from the flag evaluation
-	AttributeEvaluatedVariant = "feature_flag.evaluated_variant"
-	// AttributeEvaluatedValue trace attribute name for the value returned from the flag evaluation
-	AttributeEvaluatedValue = "feature_flag.evaluated_value"
-	// AttributeTraceName the name of the trace exported by the hook
-	AttributeTraceName = "github.com/open-feature/go-sdk-contrib/hooks/open-telemetry/pkg"
+	EventName                 = "feature_flag"
```

Comment:
Do these need to be exported?

Reply:
Ive kept this exported so that the test package can be kept separate, whilst checking for the correct value